src/utils/number_utils.ts

```diff
@@ -0,0 +1,19 @@
+export const numberUtils = {
+    // from MDN https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+    randomNumberInclusive: (minimumSpecified: number, maximumSpecified: number): number => {
+        const min = Math.ceil(minimumSpecified);
+        const max = Math.floor(maximumSpecified);
+        return Math.floor(Math.random() * (max - min + 1)) + min; // The maximum is inclusive and the minimum is inclusive
+    },
+    // creates a random number of desired length by stringing together
+    // random integers from 1-9, guaranteeing the
+    // the .toString().length of the return value to equal to numberLength parameter
+    randomNumberOfLength: (numberLength: number): number => {
```

Comment:
I know there are cooler and faster ways to achieve this (i.e. https://stackoverflow.com/a/3437180) but I wanted to guarantee that `.toString().length` = 10 so that we could correctly grab the following `timestamp` even in 300 years when `unixTimestampInSeconds.toString().length` is 11 😉 

![image](https://user-images.githubusercontent.com/210914/85361431-bfa9e080-b4d0-11ea-8a59-8f332592cb57.png)